app/HutchManager/Constants/RegistrationOptions.cs

```diff
@@ -0,0 +1,8 @@
+namespace HutchManager.Constants;
+
+public class RegistrationOptions
+{
+  public const string UserAccounts = "UserAccounts";
```

Comment:
not sure what this is doing for us?

this should probably be in the `Config/` folder with the other Options Models?